cypress/integration/customizer/general/buttons.spec.js

```diff
@@ -0,0 +1,115 @@
+let colors = {
```

Comment:
Colors could be stored inside a fixture file instead of declaring as a variable.